src/runtime_src/core/tools/xbutil2/EventTraceConfig.h

```diff
@@ -0,0 +1,339 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2025 Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
```

Comment:
I am not positive pragma once works on Linux. For consistency I think headers should be using plain old include guards.